cli/util/git.go

```diff
@@ -5,12 +5,16 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"regexp"
 	"strings"
 	"unicode"
 
 	"github.com/hashicorp/go-version"
 )
 
+// MinCommitHashLength is the minimum hash length required for installation.
```

Comment:
Why? If we notice "installation" here, this const must be in `install` package. `util` packages does not know something about installation, or packing or anything else.

Reply:
I can't do this cause install use util package and util don't use install. If i change this i will get import cycle, cause util will get install.MinCommitHashLength, so i will have to add install package in util import.

Reply:
How does `util` package know that this variable is used only for installation? What does prevent me from using this const in other packages for different purposes? This will make this comment inaccurate.
As an option, make this constant not install-specific. Or make the length requirement is installation constant, and only hash format checking - common functionality.